metadeploy/api/push.py

```diff
@@ -150,3 +153,18 @@ async def notify_org_result_changed(result):
     if await get_set_message_semaphore(channel_layer, message):
         logger.info(f"Sending message {message}")
         await channel_layer.group_send(group_name, message)
+
+
+async def notify_org_finished(job_id, error=None):
+    if error:
+        type_ = "SCRATCH_ORG_ERROR"
+    else:
+        type_ = "SCRATCH_ORG_CREATED"
+
+    message = {"type": type_, "payload": {}}
```

Comment:
@wlonk We'll want to return some info about the newly-created org, and likely also the randomly-generated UUID mentioned here? https://trello.com/c/oq68AlZm/166-i-want-to-access-an-installation-job-run-on-a-newly-created-scratch-org